src/components/site/flag/patterns/earlyAccess/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+import { earlyAccess } from './early-access.module.scss';
+import Flag from '../..';
+
+function EarlyAccess({ className }) {
+  const { t } = useTranslation();
+
+  return (
+    <Flag className={classnames(className, earlyAccess)} color="#f80">
+      {t('early_access')}
```

Comment:
using the complete string line-breaks correctly by having `width: min-content` make the element as small as possible which results in breaking.